### PR TITLE
simplify global properties

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

**Comment by CodingCarter** [comment]
You should add what problem this fixes and make a point as to why this PR should be merged.

**Comment by ry** [comment]
@CodingCarter This is to reduce indirection in the initialization process. It's less code.

File: cli/rt/22_form_data.js [hunk 1]
```diff
@@ -1,21 +1,19 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 
 ((window) => {
-  const blob = window.__bootstrap.blob;
-  const domFile = window.__bootstrap.domFile;
```
Comment (bartlomieju) [inline cli/rt/22_form_data.js:5]: I've changed my mind, I don't like this change - now this file implictly depends on both `Blob` and `File` - I much more prefer that those "deps" are listed at the top of the file.

Comment (bartlomieju) [reply cli/rt/22_form_data.js:5]: @ry we should hold of this change until we actually factor out op crates. Right now it's just unnecessarily complicating the setup by making dependencies implicit.

Comment (ry) [reply cli/rt/22_form_data.js:5]: ok fair enough. I will back out the controversial parts of this patch.

**Review by bartlomieju — APPROVED** [review]
LGTM

**Pull request merged**